Formula/gitlab-ci-multi-runner.rb

```diff
@@ -43,8 +43,10 @@ def install
 
     cd dir do
       resource("prebuilt-x86_64.tar.xz").stage do
-        system *%W[go-bindata -pkg docker -nocompress -nomemcopy -nometadata
-                   -o #{dir}/executors/docker/bindata.go prebuilt-x86_64.tar.xz]
+        system "go-bindata", "-pkg", "docker", "-nocompress", "-nomemcopy",
+                             "-nometadata", "-o",
```

Comment:
No, in this case I think it's fine, given `go-bindata` is the executable and `-pkg` is the first "true" argument.